spec/acceptance/courses_request_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 require "rspec_api_documentation/dsl"
 
-RSpec.resource "Api::V1::Courses" do
+RSpec.resource "/courses" do
```

Comment:
```suggestion
RSpec.resource "Courses" do
```

And so on in all other specs.